### PR TITLE
Single video focus view

### PR DESCRIPTION
**Pull request by Andrew387**
This MR adjusts the layout of the focus view of the video to fit the design (except the point that are out of scope).

To test run `npm run dev` and go to http://localhost:3000/video/681483fb-7cc2-4c29-a8e9-4b842a89683e?hasFocusMode=true

### CODE REVIEW

**Comment by Andrew387** [comment]
I misunderstood the scope of the ticket, the portrait resolution is now supported and adjusted the target branch

File: src/components/Video/Controls/Controls.module.scss [hunk 1]
```diff
@@ -9,13 +9,24 @@
   padding-bottom: 24px;
   font-size: 17px;
 
+  &.isFocusMode {
+    position: fixed;
+    width: 100vw;
```
Comment (Runeii) [inline src/components/Video/Controls/Controls.module.scss:14]: We're mixing a lot of different values in this file. The 100vw is fine, but looking at the use of px values above + rem + the % based sizing in the media query.

Can we settle on one where possible?

Worth n

File: src/components/Video/Controls/Controls.module.scss [hunk 2]
```diff
@@ -9,13 +9,24 @@
...
+    padding-right: 1rem;
```
Comment (Runeii) [inline src/components/Video/Controls/Controls.module.scss:15]: rem is relative to the font-size of the root element (<html>), while em is relative to this element's own font-size.

As we could change the font size of this element at different breakpoints to the root element, I tend to go with the rule of thumb of – rem can be used for font sizes, em is better for layout sizing. Sometimes that doesn't make sense, but most of the time it does.

File: src/components/Video/VideoContent/VideoContent.module.scss [hunk 3]
```diff
@@ -11,12 +11,24 @@
 
     // Custom video player styling
     :global(.vjs-control-bar) {
-      max-width: 600px;
+      width: 300px;
```
Comment (Runeii) [inline src/components/Video/VideoContent/VideoContent.module.scss:14]: Doesn't this adjust the style of the video player on the rest of the website?

File: src/components/Video/VideoContent/VideoContent.module.scss [hunk 4]
```diff
@@ -87,6 +110,22 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &.isFocusMode {
+    width: 100vw;
+    overflow: visible;
+
+    @media (--tablet){
+      overflow: visible;
```
Comment (Runeii) [inline src/components/Video/VideoContent/VideoContent.module.scss:123]: No need for this, as we apply overflow: visible above

File: src/components/Video/VideoContent/VideoContent.module.scss [hunk 5]
```diff
@@ -112,9 +151,16 @@
   aspect-ratio: 16/9;
   height: 100%;
   background: var(--light-grey);
+
+  &.isFocusMode {
```
Comment (Runeii) [inline src/components/Video/VideoContent/VideoContent.module.scss:155]: Duplicate?

File: src/components/Video/VideoContent/VideoContent.module.scss [hunk 6]
```diff
@@ -131,4 +177,10 @@
   .isLoaded & {
     opacity: 0;
   }
+
+  &.isFocusMode {
```
Comment (Runeii) [inline src/components/Video/VideoContent/VideoContent.module.scss:181]: Duplicate?

File: src/pages/video/[id]/index.module.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  width:100vw;
+  height:100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.linkButton{
+  position: absolute;
+  margin-top:10px;
```
Comment (Runeii) [inline src/pages/video/[id]/index.module.scss:13]: Does this work for all resolutions?

**Comment by Andrew387** [comment]
The changes look good but I do see that after a certain width, around 860px, the text starts to overlap with the video, I think it would be good to fix this before moving the ticket forward. But nice to see the mess of the controls a bit cleaned up he

**Pull request merged**